packages/core/e2e/examples/nextjs-edge-runtime/src/utils/llm.ts

```diff
@@ -1,9 +1,18 @@
 "use server";
 // test runtime
+import { AutoTokenizer } from "@xenova/transformers";
```

Comment:
we cannot directly import, will cause bundler error because they cannot find .node file